### PR TITLE
Add Unification

### PR DESCRIPTION
**Pull request by jo-chainmasters**

### CODE REVIEW

**Comment by HeesungB** [comment]
Hi, thank you for your contribution! We highly appreciate it.
However, You have to insert about `nodeProvider` field.

**Comment by jo-chainmasters** [comment]
I added a `nodeProvider`. But im confused about the follow error:
`Chain info has unknown field`

File: cosmos/unification.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "rpc": "https://rpc1.unification.io",
+  "rest": "https://rest.unification.io",
+  "nodeProvider": {
+    "name": "Unification",
+    "email": "hello@unification.com",
+    "website":"https://unification.com/"
+  },
+  "walletUrlForStaking": "https://explorer.unification.chainmasters.ninja/wallet",
+  "bip44": {
+    "coinType": "5555"
```
Comment (HeesungB) [inline cosmos/unification.json:13]: `coinType` should be `Number Type`

File: cosmos/unification.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "bech32PrefixValAddr": "undvaloper",
+    "bech32PrefixValPub": "undvaloperpub",
+    "bech32PrefixConsAddr": "undvalcons",
+    "bech32PrefixConsPub": "undvalconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "FUND",
+      "coinMinimalDenom": "nund",
+      "coinDecimals": "9",
```
Comment (HeesungB) [inline cosmos/unification.json:27]: `coinDecimals ` should be `Number Type`

File: cosmos/unification.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+      "coinGeckoId": "unification"
+    }
+  ],
+  "feeCurrencies": [
+    {
+      "coinDenom": "FUND",
+      "coinMinimalDenom": "nund",
+      "coinDecimals": "9",
```
Comment (HeesungB) [inline cosmos/unification.json:35]: `coinDecimals ` should be `Number Type`

File: cosmos/unification.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        "low": 300,
+        "average": 400,
+        "high": 500
+      }
+    }
+  ],
+  "stakeCurrency": {
+    "coinDenom": "FUND",
+    "coinMinimalDenom": "nund",
+    "coinDecimals": "9",
```
Comment (HeesungB) [inline cosmos/unification.json:47]: `coinDecimals ` should be `Number Type`

**Comment by HeesungB** [comment]
And your json file name should be changed to `FUND-MainNet.json`

**Comment by HeesungB** [comment]
There is one more thing to do.
Please add `chainSymbolImageUrl`

 - chainSymbolImageUrl: Image URL of the chain.
   - https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/{chain-identifier}/{file-name}.png
   - Please modify modify the chain-identifier and file-name from the link above and upload it.

**Comment by jo-chainmasters** [comment]
Cool. Thanks :)

**Pull request merged**